apps/api/src/config/database-config.js

```diff
@@ -0,0 +1,43 @@
+import { loadEnvironment } from 'planning-inspectorate-libs';
+
+loadEnvironment(process.env.NODE_ENV);
+
+export default {
+	development: {
```

Comment:
The values cam change based on the NODE_ENV var and `loadEnvironment` does that for you. I don't think you need to replicate them like this for each env.